fv3config/fv3run/_native.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import contextlib
+import resource
+import subprocess
+import os
+import tempfile
+import warnings
+import yaml
+from .._config import write_run_directory, _get_n_processes, _write_config_dict
+from ._gcloud import _copy_directory, _copy_file
```

Comment:
It seems a little odd to import a private method from a private module. Perhaps `_copy_directory` shouldn't be underscored.

Reply:
Okay. I always thought the `_` meant private to a module, rather than package-private, but it's all just a convention with no teeth anyway.